homeassistant/components/alexa/capabilities.py

```diff
@@ -1113,7 +1113,7 @@ class AlexaSecurityPanelController(AlexaCapability):
     https://developer.amazon.com/docs/device-apis/alexa-securitypanelcontroller.html
     """
 
-    supported_locales = {"en-AU", "en-CA", "en-IN", "en-US"}
+    supported_locales = {"en-AU", "en-CA", "en-IN", "en-US", "it-IT"}
```

Comment:
```suggestion
    supported_locales = {"de-DE", "en-AU", "en-CA", "en-GB", "en-IN", "en-US", "es-ES", "es-MX", "es-US", "fr-CA", "fr-FR", "it-IT", "ja-JP", "pt_BR"}
```

The documentation has been updated to support the following locales for this capability.